### PR TITLE
external links toggle feature specs

### PR DESCRIPTION
**Pull request by banukutlu**
Part of #759

### CODE REVIEW

File: spec/features/availability_spec.rb [hunk 1]
```diff
@@ -227,7 +227,7 @@
           expect(page).not_to have_selector 'button', text: /View Less/
           expect(page).to have_xpath './/tbody/tr', count: 5
           click_button('View More')
-          sleep 1 # let collapse animation finish and wait for it to re-collapse
+          # sleep 1 # let collapse animation finish and wait for it to re-collapse
```
Comment (nicktiberi) [inline spec/features/availability_spec.rb:230]: Should we just remove this instead of commenting it out?

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -97,3 +97,6 @@
 Capybara.configure do |config|
   config.javascript_driver = :chrome_headless # This is slower
 end
+
+# Disable CSS animations which slows down tests
+Capybara.disable_animation = true
```
Comment (nicktiberi) [inline spec/rails_helper.rb:102]: Nice! 🚀

File: spec/features/availability_spec.rb (hunk 1 above)
Comment (banukutlu) [reply spec/features/availability_spec.rb:230]: ahh I thought I removed it 🤦🏻  yes we better remove, will do :) good catch

File: spec/rails_helper.rb (hunk 2 above)
Comment (banukutlu) [reply spec/rails_helper.rb:102]: isn't it? also read that css animations and transitions might be causing flakey tests so I tried disabling them globally, it also let to get rid of those `sleep 1` in the other specs as well.

**Review by nicktiberi — APPROVED** [review]
💯

File: spec/rails_helper.rb (hunk 2 above)
Comment (nicktiberi) [reply spec/rails_helper.rb:102]: Mayyybeee this could fix the flaky spec in myaccount? 🤞 👀

Comment (banukutlu) [reply spec/rails_helper.rb:102]: yes we should try this over there 🤞🏻

**Pull request merged**